charts/coolify/templates/coolify-app-ingress.yaml

```
@@ -0,0 +1,36 @@
+metadata:
+  name: {{ include "coolify.coolifyApp.fullname" . }}-ingress
+  namespace: {{ include "coolify.namespace" . }}
+  labels:
+    {{- include "coolify.labels" (merge (dict "componentName" "core") .) | nindent 4 }}
+    app: coolify
+    component: core
+  annotations:
+    {{- toYaml .Values.ingress.annotations | nindent 4 }}
+spec:
+  {{- if .Values.ingress.className }}
+  ingressClassName: {{ .Values.ingress.className }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)